tests/axios.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test } from '../src/fixtures/baseFixtures';
+import  axios from 'axios';
+import { wrapper } from 'axios-cookiejar-support';
+import { CookieJar } from 'tough-cookie';
+
+test.describe('Books Collection User Actions with axios', async () => {
+    let client: Axios;
+    let userId: string;
+    let token: string;
+
+
+    test.beforeAll('Login User and save user data', async () => {
+        const jar = new CookieJar();
+
+        client = wrapper(axios.create({
+            baseURL: process.env.LOGIN_URL,
+            validateStatus: function (status) {
+                return status >= 200 && status < 300;
+            },
+            jar
+        }));
+
+        const loginData = await client.post('https://demoqa.com/Account/v1/Login', {

```

Comment:
you've setupped base Url in line 16, why you put here full url?
```suggestion
        const loginData = await client.post('/Account/v1/Login', {

```